go.mod

```diff
@@ -3,9 +3,13 @@ module sda-pipeline
 go 1.14
 
 require (
+	github.com/aws/aws-sdk-go v1.31.13
 	github.com/elixir-oslo/crypt4gh v0.0.2
+	github.com/go-ini/ini v1.57.0 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/lib/pq v1.6.0
+	github.com/minio/minio-go v6.0.14+incompatible
+	github.com/minio/minio-go/v6 v6.0.56
```

Comment:
Why is minio included here if it is not used?

Reply:
Fixed